src/App.css

```diff
@@ -9,22 +9,29 @@
 .App {
   text-align: left;
   display: grid;
-  grid-template-rows: [header] 80px [main] auto [footer] 80px;
+  height: 100vh;
+  grid-template-rows: [header] 80px [main] auto;
+  background-image: url("./images/background.png");
+  background-size: cover;
+  color: #ffff;
 }
 
 .Header {
   font-family: "Work Sans", sans-serif;
   font-size: 40px;
   grid-row: header;
+  margin: 20px;
 }
 
 .Main {
   grid-row: main;
+  overflow-y: scroll;
 }
 
 .Footer {
   grid-row: footer;
-  display: flex;
-  justify-content: space-around;
-  align-items: center;
+  margin: auto 0;
+  position: absolute;
+  top: 600px;
+  width: 100vw;
```

Comment:
something is off with the scrolling. you can scroll further than the image and then have a white background behind your footer
![Screenshot 2021-04-27 at 10 11 03](https://user-images.githubusercontent.com/78050731/116208147-e2d09c80-a740-11eb-8ea8-1c163653ae1a.png)